0.19/Dockerfile

```diff
@@ -25,12 +25,25 @@ RUN set -ex \
       gpg --batch --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" ; \
     done \
   && curl -SLO https://bitcoin.org/bin/bitcoin-core-${BITCOIN_VERSION}/SHA256SUMS.asc \
-  && curl -SLO https://bitcoin.org/bin/bitcoin-core-${BITCOIN_VERSION}/bitcoin-${BITCOIN_VERSION}-x86_64-linux-gnu.tar.gz \
+  && curl -SLO https://bitcoin.org/bin/bitcoin-core-${BITCOIN_VERSION}/bitcoin-${BITCOIN_VERSION}-${ARCH}-linux-gnu.tar.gz \
   && gpg --verify SHA256SUMS.asc \
-  && grep " bitcoin-${BITCOIN_VERSION}-x86_64-linux-gnu.tar.gz\$" SHA256SUMS.asc | sha256sum -c - \
+  && grep " bitcoin-${BITCOIN_VERSION}-${ARCH}-linux-gnu.tar.gz\$" SHA256SUMS.asc | sha256sum -c - \
   && tar -xzf *.tar.gz -C /opt \
-  && rm *.tar.gz *.asc
+  && rm -rf /opt/bitcoin-${BITCOIN_VERSION}/bin/bitcoin-qt \
+  && mv /opt/bitcoin-${BITCOIN_VERSION}/bin/* /usr/local/bin \
+  && mv /opt/bitcoin-${BITCOIN_VERSION}/lib/* /usr/local/lib \
+  && mkdir -p /home/bitcoin/.bitcoin
+
+FROM $ARCH/debian:stable-slim
```

Comment:
Why this arch is needed? Docker build will pull the correct image from the hub for the current arch